python_modules/dagster/dagster/_core/execution/submit_asset_runs.py

```diff
@@ -88,9 +88,7 @@ def _get_job_execution_data_from_run_request(
         op_selection=None,
     )
 
-    selector_id = hash_collection(pipeline_selector)
-
-    if selector_id not in run_request_execution_data_cache:
```

Comment:
Flagging because it seems different from rest of PR but I trust you know what you're doing

Reply:
ya im not sure why we were using `hash_collection` here instead of just keying by tuple, maybe it wasn't a tuple to begin with? either way just using the tuple directly as the key should be equivalent